hcert_spec.md

```diff
@@ -165,17 +177,78 @@ However, regardless if an Issuer decides to use HSMs or not, a key roll-over sch
+
+Memberstates are required to keep a public register of these certificates at a stable URL.
+
+Memberstates may then bilaterally exchange CSCA certificates with a number of other States, verify these bilaterally and thus compile their own lists of CSCA certificates: a (MS specific) Master List. 
+
+These CSCA certificates regularly sign the Document Signing Certificates (DSC) used in day to day operations. Memberstates will each will maintain a public register of the DSC certificates that is kept current.
+
+Other memberstates must regularly fetch these list of DSC certificates and cryptographically verify these against the CSCA certificates (that they have verified by other, non-digital, means).  
+
+The resulting list of DSC certificates then provides the acceptable public keys (and the corresponding KIDs) that verifiers can use to validate the signature on the CWT in the Qr code. 
+
+Verifiers should fetch update so this list regularly. Verifiers are expected to tune the format to this list for their own national setting; and the file format of this, internal, trusted list may vary, e.g. it can be a plain JWKS like https://github.com/ehn-digital-green-development/hcert-testdata/blob/main/testdata/jwks.json or something specific to the technology used.
+
+## Differences with the ICAO MasterList system for passports
+
+While patterned on best practices of the ICAO Ml - there are a number of simplifications made in the interest of speed (and recognising the fact that the EU Regulation for EHN is sharply limited in time and scope).
+
+* A Member-state may submit multiple CSCA certificates
+* A CSCA certificate may also be used --and published as-- a DSC.
+* The DSC (key usage) validity period may be set to any length not exceeding the CSCA.
+* The DSC certificate MAY contain policy identifers that are EHN specific.
+
+## Secretariat
+
+In order to alleviate the burden of countries during the initial phase -- there shall be a secretarial service that will:
+
+* Maintain a list of operational and legal contacts for each member-state to further orderly management of this health specific set of master lists.
+* Maintain a public 24x7 an incident/security contact point.
+* Maintain a public list of URLs with the most up to date CSCA lists for each member-state.
+* Maintain a public  list of URLs with the most up to date DSC lists for each member-state.
+* Maintain a public  single, aggregated, list of all CSCAs, that is updated daily.
+* Maintain a public single, aggregated, list of all DSAs, that is updated daily.
+* Provides MS with a secure (i.e. integrity protected) manner by which to provide the Secretariat with the MS its CSCA and DSC lists (CIRBAC, t.b.c)
+* Shall validate the DSCs against the CSCA prior to publication.
+* MAY sign the aggregated list.
```

Comment:
I think this is taken care of in the previous bullets? I suggest to strike it out.

Reply:
Ok - just strike it.

Reply:
done